sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -145,6 +145,7 @@ export interface Sender {
     scheduleMessage(scheduledEnqueueTimeUtc: Date, message: ServiceBusMessage, options?: OperationOptions): Promise<Long>;
     scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: ServiceBusMessage[], options?: OperationOptions): Promise<Long[]>;
     send(message: ServiceBusMessage, options?: OperationOptions): Promise<void>;
+    send(messages: ServiceBusMessage[], options?: OperationOptions): Promise<void>;
```

Comment:
This is going to re-open the discussion of https://github.com/Azure/azure-sdk-for-js/pull/8470#discussion_r413173631

Should this be sendBatch? Or should we change the sendBatch to send and have just 3 overloads for send?

Reply:
I don't mind just making them all send() overloads (do you want that in this PR?). After the user study I realized that having them as separate methods just confuses people.

Reply:
I'll create an issue for this and we can discuss this there. Definite negative - if we migrate the 'batch' part into `send()` then we're different than EventHubs.

Reply:
Filed as issue https://github.com/Azure/azure-sdk-for-js/issues/8571

Reply:
For this PR, I would say that we use the same name for batch and array so that we stick to the spirit of what was discussed in https://github.com/Azure/azure-sdk-for-js/pull/8470#discussion_r413173631

Whether that means `send(message|messages|batch, options)` or
```js
send(message, options)
sendBatch(messages|batch, options)
```
is up to you

Reply:
Ha, you know which one I'm going to pick!

Reply:
Done and with no surprises at all - I'm using send() for EVERYTHING now.